### ISSUE TITLE
Fortios backup

### ISSUE THREAD

**Issue by wolf2668** [ytti/oxidized#1386]
Hi,

Everytime my equipements are backuped, my fortinet has a new version because password and private-key changed.
But, in fortios.rb, there is : 

```
cmd :secret do |cfg|
    # ENC indicates an encrypted password, and secret indicates a secret string
    cfg.gsub! /(set .+ ENC) .+/, '\\1 <configuration removed>'
    cfg.gsub! /(set .*secret) .+/, '\\1 <configuration removed>'
    # A number of other statements also contains sensitive strings
    cfg.gsub! /(set (?:passwd|password|key|group-password|auth-password-l1|auth-password-l2|rsso|history0|history1)) .+/, '\\1 <configuration removed>'
    cfg.gsub! /(set private-key).*-+END ENCRYPTED PRIVATE KEY-*"$/m, '\\1 <configuration removed>'
    cfg.gsub! /(set ca ).*-+END CERTIFICATE-*"$/m, '\\1 <configuration removed>'
    cfg.gsub! /(set csr ).*-+END CERTIFICATE REQUEST-*"$/m, '\\1 <configuration removed>'
    cfg.gsub! /(Cluster uptime:).*/, '\\1 <stripped>'
    cfg
  end
```

I don't know why it is working with other equipments and not fortinet... Do you have any idea how to fix it ?

EDIT : Same thing with Cisco SMB (System Up Time)

Thanks !

**Comment by wk** [comment]
Do you have [Removing secrets](https://github.com/ytti/oxidized/blob/fcb0e5e462e9b3d11d0c228aafd5a853b202746f/docs/Configuration.md#removing-secrets) enabled in your oxidized configuration? Without it, the block you've quoted would not come into effect.

Ditto for `ciscosmb.rb`, however there - this is caused by a bug where the uptime is only scrubbed if secret removal is also enabled; #1388 will address this issue.

**Comment by wk** [comment]
Closed by #1388

**Comment by wolf2668** [comment]
You're right, this line doesn't exist in my config file.
It will work better :)

Thanks !
